src/ci/docker/run.sh

```diff
@@ -264,6 +264,7 @@ docker \
   --env RUST_CI_OVERRIDE_RELEASE_CHANNEL \
   --env CI_JOB_NAME="${CI_JOB_NAME-$IMAGE}" \
   --env BASE_COMMIT="$BASE_COMMIT" \
+  --env DIST_TRY_BUILD="${DIST_TRY_BUILD}" \
```

Comment:
you don't need to specify a value if you're just passing it through
```suggestion
  --env DIST_TRY_BUILD \
```